upup/pkg/fi/cloudup/openstacktasks/instance.go

```diff
@@ -151,12 +179,23 @@ func (_ *Instance) ShouldCreate(a, e, changes *Instance) (bool, error) {
 	return a == nil, nil
 }
 
+func makeServerName(e *Instance) string {
+	hash := utils.RandomString(6)
+	// FIXME: Must ensure 63 or less characters
+	// replace all dots and _ with -, this is needed to get external cloudprovider working
+	stripped := strings.Replace(fi.StringValue(e.ClusterName), ".k8s.local", "", -1)
```

Comment:
As for the limit of 63 characters for the name, can't we also just remove the cluster name from the instances, as we already have the k8s tags in place?

Reply:
it is removed in new format